### PR TITLE
Return null instead when no isoline info

### PR DESCRIPTION
**Pull request by rafatower**
@ethervoid and/or @iriberri please review. Thanks!

this should go into next release (more sooner than later).

### CODE REVIEW

File: server/extension/sql/80_isolines_helper.sql [hunk 1]
```diff
@@ -93,20 +93,23 @@ RETURNS SETOF cdb_dataservices_server.isoline AS $$
...
 
     result = []
     for r in data_range:
 
-      # -- TODO encapsulate this block into a func/method
-      locations = isolines[r] + [ isolines[r][0] ] # close the polygon repeating the first point
-      wkt_coordinates = ','.join(["%f %f" % (l['lon'], l['lat']) for l in locations])
-      sql = "SELECT ST_MPolyFromText('MULTIPOLYGON((({0})))', 4326) as geom".format(wkt_coordinates)
-      multipolygon = plpy.execute(sql, 1)[0]['geom']
+      if len(isolines[r]) >= 3:
```
Comment (ethervoid) [inline server/extension/sql/80_isolines_helper.sql]: Why 3? could you please put a comment to explain this a bit?

Comment (rafatower) [inline server/extension/sql/80_isolines_helper.sql]: the minimum polygon requires 3 points to be defined

**Comment by iriberri** [comment]
LGTM

**Comment by rafatower** [comment]
@iriberri could you please take care of releasing this? (all changes lie in the server extension side of things)

**Comment by iriberri** [comment]
Sure, I'm gonna add the required function changes in the branch I have to bump the versions.

**Comment by iriberri** [comment]
Done

**Pull request merged**